src/modules/apps/deploy.ts

```diff
@@ -34,7 +33,7 @@ const deployRelease = async (app: string): Promise<void> => {
   if (vendor !== account) {
     const canSwitchToVendor = await promptConfirm(switchToVendorMessage(vendor))
     if (!canSwitchToVendor) {
-      throw new UserCancelledError()
+      return
```

Comment:
In this case we would have a problem on just returning, becase the caller function would just print `deployed successfully`
Maybe we could return null on cancel and the called would check if before logging?